### ISSUE TITLE
Downloading time

### ISSUE THREAD

**Issue by verakai** [kvos/CoastSat#98]
Good morning,

Congratulations to the CoastSat team for your effort in putting this project together and sharing it with all of the community, it is super detailed and it is a pleasure to read such a well documented project !

My issue is that it is taking endless time, +1~2 hrs and counting to download a reduced polygon of approximately 0.25 km2, S2 image, and data of only one day of timeframe, so I'm actually attempting to download only one image.

I was reading a latter issue indicating that the download time for the polygon in the example.py file was taking 3.5hrs, so, being this a considerable smaller polygon and a reduce time frame, I think it should take a considerable less time to download. (I'm only attempting to test the downloading part)

My question is, if the original example's polygon measures 5.87 km2 approx, taking 3.5 hrs to download, and the polygon I'm using measure 0.25 km2 approx, is it normal that it is taking more than 2 hours downloading ?

I am trying to download both in my local machine (my cable is 25 mb/s) as in my google Colab notebook.

Thank you for any input, and thank you again for having uploaded this project !

**Comment by verakai** [comment]
Hi,

The issue was resolved in both Colab and my local environment. On Colab I had to update only the following packages to the versions in the environment.yml:

!pip3 install matplotlib==3.0.3
!pip3 install pillow==6.2.1
!pip3 install earthengine-api==0.1.173
!pip3 install gdal==2.3.3
!pip3 install pandas==0.24.2
!pip3 install geopandas==0.4.1

And the downloading part worked.

On my local machine, I removed the "coastsat" environment I had created, and installed it again with all the packages in environment.yml. I was thinking something was wrong there, and I would possibly have to update it, as I did in Colab, but it wasn't necessary. This time the environment was correctly setup.

Bottom line, be sure the version of the packages as suggested in the environment.yml file are correctly set up.

Thank you once again for this great project and how detailed it is explained !!!
